math/minuit2/inc/Minuit2/MnMigrad.h

```diff
@@ -72,8 +72,6 @@ class MnMigrad : public MnApplication {
    /// construct from FCNGradientBase + MnUserParameterState + MnStrategy
    MnMigrad(const FCNGradientBase& fcn, const MnUserParameterState& par, const MnStrategy& str) : MnApplication(fcn, MnUserParameterState(par), str), fMinimizer(VariableMetricMinimizer()) {}
 
-   MnMigrad(const MnMigrad& migr) : MnApplication(migr.Fcnbase(), migr.State(), migr.Strategy(), migr.NumOfCalls()), fMinimizer(migr.fMinimizer) {}
-
    ~MnMigrad() {}
```

Comment:
```suggestion
   MnMigrad(const MnMigrad& migr) = default;
   ~MnMigrad() {}
```

Reply:
I follow the "rule of zero" here, so no, I think we should not add an explicitly defaulted copy ctor.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c20-if-you-can-avoid-defining-default-operations-do

Reply:
Not that we _have_ to agree with such guidelines, but anyway right below, this applies in this case: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all as the destructor and the copy-assignment operators are defined.

Reply:
Ok, but making copy assignment private makes no sense here. If the class is default copyable it is also default assignable.

Reply:
I also removed the private assignment, so I think this ready to merge now.

Reply:
this is not going in the direction I suggested xD but I'm not the maintainer of Minuit, so up to @lmoneta , I don't have a strong opinion